ambry-replication/src/main/java/com.github.ambry.replication/BackupManager.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.replication;
```

Comment:
Should this  be in the ambry-cloud module? Instead of ambry-replication depending on ambry-cloud you can have ambry-cloud package depend on ambry-replication

Reply:
yes. Moving it to ambry-cloud.